packages/manager/modules/pci/src/projects/project/storages/volume-backup/list/create-volume/controller.js

```diff
@@ -0,0 +1,96 @@
+export default class VolumeBackupListCreateVolumeController {
+  /* @ngInject */
+  constructor($translate, ovhManagerRegionService, VolumeBackupService) {
+    this.$translate = $translate;
+    this.ovhManagerRegionService = ovhManagerRegionService;
+    this.volumeBackupService = VolumeBackupService;
+  }
+
+  $onInit() {
+    this.isCreating = false;
+    this.volumeSizeLimit = {
+      min: 10,
```

Comment:
Could you please use constants for the min/max initial values ?

Reply:
Yep sure we could.
But no other usage except here.

Reply:
Creating CONST variables for only one usage.
Personally i avoid.

Reply:
The idea behind using const in this case is to follow a pattern where all the initialisation values must be defined as a constant in another file, so that if one day we want to implement unit/integration tests then these test could be easier to maintain.
If you find this case too niche to be considered than it's ok.

Reply:
No, it's a good point about the testing 👍

Reply:
done